### ISSUE TITLE
Error on Delegation

### ISSUE THREAD

**Issue by propertunist** [aaroncox/vessel#59]
I previously used Vessel to delegate 2.5k steem from one account to another and just cancelled that delegation a week ago. I then started the power down process for the source account and then attempted to re-delegate half of the steem power back to the same account it was originally delegated to.

The slider in vessel shows the correct amount of SP and I set it to roughly 50% of the total. However, Vessel then presented the following error message with an option to cancel or continue:
```
available_shares >= op.vesting_shares: Account does not have enough vesting shares to delegate.
```

What to do?

**Comment by aaroncox** [comment]
Has a week actually passed? It takes a full 7 days for the SP to return to your account.

**Comment by propertunist** [comment]
yes, the week is up and the SP is in the account according to Steemit anyway.
I just tested the new version of vessel and now see:
``` 'Operation Error'. 
```

Is it a bad idea to click 'Confirm Delegation'?

**Comment by aaroncox** [comment]
I don't have a solution for you at this moment - but if you don't mind, some more information would be good to know exactly what's happening here.

- What account was this on?
- How much SP/Vests were you trying to delegate?

From what it sounds like, I don't know this is an issue with vessel, but just an issue with Steem in general (and it's confusing nature). If you want me to dig a bit deeper I just need a bit more info.

**Comment by propertunist** [comment]
Ok, no problem. the source account is david.neale and the destination is ura-soul.
The amount is: 1.254k SP -2.571M VESTS

**Comment by aaroncox** [comment]
I wonder if it's because it's powering down? I'm going to guess that you can only delegate funds that aren't currently in the process of being powered down, though I'll have to confirm.

**Comment by propertunist** [comment]
could be. i'll try redelegating when the next payout is made.

**Comment by apoloo11** [comment]
to me the same thing happened with 6 accounts they did not return to any of them the sp and they already spent more than 10 days
